<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntervalInsertWindow</class>
 <widget class="QDialog" name="IntervalInsertWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interval Input</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Max iterations</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="it_spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Polynomial degree</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="deg_spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="coef_label">
         <property name="text">
          <string>Coefficent a[0]</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="coefBox">
         <item>
          <widget class="QLineEdit" name="coef_lineEditP"/>
         </item>
         <item>
          <widget class="QLineEdit" name="coef_lineEditL"/>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="prev_next_Box">
         <item>
          <widget class="QPushButton" name="prev_pushButton">
           <property name="text">
            <string>Prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="next_pushButton">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="insert_pushButton">
         <property name="text">
          <string>Insert data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_pushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
